applications/StructuralMechanicsApplication/custom_processes/compute_rayleigh_damping_coefficients_process.cpp

```diff
@@ -0,0 +1,145 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:             BSD License
+//                                       license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/linear_solver_factory.h"
+#include "utilities/math_utils.h"
+#include "custom_processes/compute_rayleigh_damping_coefficients_process.h"
+#include "structural_mechanics_application_variables.h"
+#include "custom_strategies/custom_schemes/eigensolver_dynamic_scheme.hpp"
+#include "custom_strategies/custom_strategies/eigensolver_strategy.hpp"
+#include "solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver_with_constraints.h"
```

Comment:
I am not quite sure if it is a good idea to directly include those

Why not writing this process in python? There you could easily plug together the different modules

Reply:
Huuum, the problem is if I want to call it from c+, but I assume that it will be fine